packages/usa-range/src/index.js

```diff
@@ -0,0 +1,52 @@
+const selectOrMatches = require("../../uswds-core/src/js/utils/select-or-matches");
+const behavior = require("../../uswds-core/src/js/utils/behavior");
+
+const { prefix: PREFIX } = require("../../uswds-core/src/js/config");
+
+const RANGE_CLASSNAME = `${PREFIX}-range`;
+const RANGE = `.${RANGE_CLASSNAME}`;
+
+/**
+ * Called when range slider has optional unit data attribute.
+ * Collects necessary attributes, then constructs and appends aria-valuetext attribute for screen reader context.
+ *
+ * @param {HTMLInputElement} targetRange - The range slider input element
+ */
+const updateCallout = (targetRange) => {
+  const rangeSlider = targetRange;
+  const unit = rangeSlider.getAttribute("data-optional-unit");
+  const val = rangeSlider.value;
+  const max = rangeSlider.getAttribute("max");
+  const callout = `${val} ${unit || ""} of ${max}`;
```

Comment:
Wondering if we could add multi-lingual support here by adding an optional custom text data attribute for "of"

Reply:
I went ahead and added this in [cbfe406](https://github.com/uswds/uswds/pull/5472/commits/cbfe406e881c73ca35466a6b681b468c31425e41)

The language for the numbers will still rely on users system settings, but this will allow devs to update the language as expected!